templates/mopro-example-app/core/circuits/halo2-fibonacci/README.md

```diff
@@ -0,0 +1,24 @@
+# Halo2 Examples
+
+This repo includes a few simple examples to illustrate how to write circuit in Halo2.
+
+## Instruction
+
+Compile the repo
+
+```
+cargo build
```

Comment:
this doesn't work in this directory
```sh
error[E0432]: unresolved import `halo2_examples`
 --> core/circuits/halo2-fibonacci/src/main.rs:9:5
  |
9 | use halo2_examples::{FinbonaciCircuit, write_keys, write_srs};
  |     ^^^^^^^^^^^^^^ use of undeclared crate or module `halo2_examples`

```

Reply:
maybe outdated or need to be run in other directory?

Reply:
It was, fixed it, thank you for pointing out!